docs/modules/ROOT/pages/resource-configuration.adoc

```diff
@@ -0,0 +1,37 @@
+= Configuring Container Resources
+
+:description: pass:q[You can configure resources for Hazelcast, sidecar agent and management center containers.]
+
+{description}
+
+Detailed documentation about resources can be found link:https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/[in Resource Management for Pods and Containers documentation of Kubernetes].
+
+== Configuring Resources for Hazelcast
+
+You can configure resources for Hazelcast containers using the `resources` field of the CRD spec.
```

Comment:
```suggestion
You can configure resources for Hazelcast containers using the `resources` field of the CRD specification.
```

Reply:
`spec` is a field, so it is better to mention as `spec` IMO.

Reply:
OK, could we say "the CRD `spec` field" just so it is clear? 

Although that is two "fields" in close proximity....are they both called fields? Could we call `resources` a sub-field? My knowledge of K8s terminology is not good. 

Alternatively, just put the ticks around spec (/`spec/`) - that is probably sufficient.

Reply:
> Although that is two "fields" in close proximity....are they both called fields? Could we call resources a sub-field?

`resources` is located under `spec`, so it is a kind of sub-field.

> Alternatively, just put the ticks around the spec.a That is probably sufficient.

Done 👍